<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPane_Visuals</class>
 <widget class="QWidget" name="ControlPane_Visuals">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="controls">
     <item>
      <widget class="FoldingGroupBox" name="viewportGroup">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Viewport</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="verticalWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="1" column="0">
              <widget class="QLineEdit" name="widthLineEdit">
               <property name="font">
                <font>
                 <family>Consolas</family>
                 <pointsize>8</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="heightLineEdit">
               <property name="font">
                <font>
                 <family>Consolas</family>
                 <pointsize>8</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Resolution</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>X</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QFrame" name="bgcolorFrameButton">
               <property name="minimumSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>24</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: red</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::Panel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="lineWidth">
                <number>3</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="bgcolorLabel">
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Background Color</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="FoldingGroupBox" name="colormapGroup">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Colormap</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="verticalWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QComboBox" name="colormapSelector">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="reverseCheckBox">
             <property name="text">
              <string>Reverse colormap</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="outofrangeCheckBox">
             <property name="text">
              <string>Out-of-range colors</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QFrame" name="abovecolorFrameButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>24</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: white</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::Panel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="lineWidth">
                <number>3</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="abovecolorLabel">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Above-Range Color</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QFrame" name="belowcolorFrameButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>24</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: black</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::Panel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="lineWidth">
                <number>3</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="belowcolorLabel">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Below-Range Color</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="FoldingGroupBox" name="widgetsGroup">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Widgets</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="verticalWidget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="horizontalSpacing">
              <number>8</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="eyeLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>Segoe Fluent Icons</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string></string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="orientationLabel">
               <property name="text">
                <string>Orientation</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" alignment="Qt::AlignHCenter">
              <widget class="QCheckBox" name="scalarbarvisCheckBox">
               <property name="text">
                <string/>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1" alignment="Qt::AlignHCenter">
              <widget class="QCheckBox" name="scalarbarmoveCheckBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="1" alignment="Qt::AlignHCenter">
              <widget class="QCheckBox" name="orientationmoveCheckBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="0" alignment="Qt::AlignHCenter">
              <widget class="QCheckBox" name="orientationvisCheckBox">
               <property name="text">
                <string/>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="scalarbarLabel">
               <property name="text">
                <string>Scalar bar</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="moveLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>Segoe Fluent Icons</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string></string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_9">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="FoldingGroupBox" name="glyphsGroup">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Glyphs</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="verticalWidget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QCheckBox" name="opacityCheckBox">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="toolTip">
              <string>WARNING: Seriously impacts performance!</string>
             </property>
             <property name="text">
              <string>Opacity ⚠</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="opacitycontrolFrame">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Clipped opacity overrides masked opacity</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <property name="horizontalSpacing">
               <number>2</number>
              </property>
              <item row="1" column="0">
               <widget class="QLabel" name="orientationLabel_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>4</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Clipped&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="maskopacityvalueLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>3</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>100%</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSlider" name="clipopacitySlider">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>9</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximum">
                 <number>20</number>
                </property>
                <property name="singleStep">
                 <number>1</number>
                </property>
                <property name="pageStep">
                 <number>4</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSlider" name="maskopacitySlider">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>9</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="mouseTracking">
                 <bool>false</bool>
                </property>
                <property name="maximum">
                 <number>20</number>
                </property>
                <property name="singleStep">
                 <number>1</number>
                </property>
                <property name="pageStep">
                 <number>4</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="scalarbarLabel_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>4</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Masked&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="clipopacityvalueLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>3</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>0%</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FoldingGroupBox</class>
   <extends>QGroupBox</extends>
   <header location="global">zoo.ui.foldinggroupbox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
